### PR TITLE
Check if averagePrice is undefined, if it is undefined then get the p…

### PR DESCRIPTION
**Pull request by kchowhan**
…rice from totalPrice

### CODE REVIEW

File: src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954 [hunk 1]
```diff
@@ -83,6 +83,12 @@ try{
           u.data.url = location.protocol + "//" + u.data.url;
         }
 
+        if (typeof b['entity.hotels.results.results.0.rooms.0.averagePrice.amount'] != 'undefined' && b['entity.hotels.results.results.0.rooms.0.averagePrice.amount']) {
+          u.data.base_price = b['entity.hotels.results.results.0.rooms.0.averagePrice.currency'] + Math.round(b['entity.hotels.results.results.0.rooms.0.averagePrice.amount']);
+        } else {
+          u.data.base_price = b['entity.hotels.results.results.0.rooms.0.totalPrice.currency'] + Math.round(b['entity.hotels.results.results.0.rooms.0.totalPrice.amount']);
```
Comment (trungnguyenexp) [inline src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954]: I didn't find this in utag_data: 'entity.hotels.results.results.0.rooms.0.totalPrice.currency' on HSR page; therefore, this block won't get hit.

File: src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954 [hunk 2]
```diff
@@ -83,6 +83,12 @@ try{
           u.data.url = location.protocol + "//" + u.data.url;
         }
 
+        if (typeof b['entity.hotels.results.results.0.rooms.0.averagePrice.amount'] != 'undefined' && b['entity.hotels.results.results.0.rooms.0.averagePrice.amount']) {
+          u.data.base_price = b['entity.hotels.results.results.0.rooms.0.averagePrice.currency'] + Math.round(b['entity.hotels.results.results.0.rooms.0.averagePrice.amount']);
```
Comment (trungnguyenexp) [inline src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954]: I didn't find this in utag_data: 'entity.hotels.results.results.0.rooms.0.averagePrice.amount' on HSR page; therefore, this block won't get hit.

File: src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954 (hunk 1 above)
Comment (kchowhan) [reply src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954]: HSR team is fixing it at the moment and should be in trunk soon. @trungnguyenexp

File: src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954 [hunk 3]
```diff
@@ -83,6 +83,12 @@ try{
...
+        } else {
+          u.data.base_price = b['entity.hotels.results.results.0.rooms.0.totalPrice.currency'] + Math.round(b['entity.hotels.results.results.0.rooms.0.totalPrice.amount']);
+        }
+
         u.data.base_price = b['entity.hotels.results.results.0.rooms.0.averagePrice.currency'] + Math.round(b['entity.hotels.results.results.0.rooms.0.averagePrice.amount']);
```
Comment (demadison) [inline src/profile/main/tag/Google Hotels Price Accuracy Tracking - #1954]: might want to remove this line

**Pull request merged**